config.docker.yml

```diff
@@ -1,6 +1,6 @@
 public_dir: ./public
 package_paths:
-  - ./packages/package-storage
+  - /packages
```

Comment:
Let me explain why was there a reference to the `package-storage`.

In integrations we mount the second path for packages, so that we have:

```
./packages/package-storage
./packages/integrations
```
so that all packages are stored in the same `/packages` directory. Using a single directory prevents from simply loading another path with packages without replacing existing ones.

If you plan to change the current behavior here, I believe you need to update the testing/environment in integrations.

Reply:
My assumption was this keeps working as in https://github.com/elastic/integrations/blob/master/testing/environments/snapshot.yml#L47 you overwrite the default config?

Reply:
Updated to use `/packages/package-registry` as default and added a note to the dockerfile and config what users should do about it.